tests/unit/disco/containers/TestDiscoPane.js

```diff
@@ -163,14 +178,60 @@ describe('AddonPage', () => {
       const { handleGlobalEvent } = helpers.mapDispatchToProps(dispatch);
       const payload = { id: 'whatever' };
       handleGlobalEvent(payload);
-      expect(dispatch.calledWith({ type: INSTALL_STATE, payload })).toBeTruthy();
+      sinon.assert.calledWith(dispatch, { type: INSTALL_STATE, payload });
     });
 
-    it('is empty when there is no navigator', () => {
-      const configStub = {
-        get: sinon.stub().returns(true),
-      };
-      expect(helpers.mapDispatchToProps(sinon.spy(), { _config: configStub })).toEqual({});
+    it('does not pass handleGlobalEvent when on the server', () => {
+      const dispatch = sinon.stub();
+      const configSource = { server: true };
+      const configStub = { get: (key) => configSource[key] };
+      expect(helpers.mapDispatchToProps(dispatch, { _config: configStub }))
+        .toEqual({ dispatch });
+    });
+  });
+
+  describe('constructor', () => {
+    it('gets discovery results when results are 0 length', () => {
```

Comment:
This phrasing is a bit awkward. Maybe `gets discovery results if results are empty`?